cookiecutter/{{cookiecutter.github_repo_name}}/CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contributing to {{cookiecutter.title}}
+
+## Build
+
+You can use `build` to simply build the source and binary distribution:
+
+```sh
+python -m pip install build
+python -m build
+```
+
+## Local development
```

Comment:
This is a bit misleading in this scope, because for local development in rust, these instructions are not relevant. As suggested, moving this to a section dedicated to the python library would be clearer.

Reply:
Replied in https://github.com/frequenz-floss/frequenz-repo-config-python/pull/61#discussion_r1238258821, let's continue the conversation about if we should add a Python section **now** there so we don't duplicate it.